spec/e2e/pages/short-form/you-name.coffee

```diff
@@ -2,6 +2,7 @@ AngularPage = require('../angular-page').AngularPage
 
 class Name extends AngularPage
   constructor: ->
+    super()
```

Comment:
what does the super() do in these?

Reply:
I just tried to remove it and our e2e tests failed, I think it's because we've updated our ES version and now it's a requirement: https://github.com/jashkenas/coffeescript/issues/4547